pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -589,6 +592,18 @@
     ==
     (star next)
   ==
+::  +scry-to-mime: scry a path, converting to mime
+::
+++  scry-to-mime
+  |=  [=path rof=roof]
+  ^-  (unit mime)
+  ?~  path  ~
+  ?~  bem=(de-beam t.path)  ~
+  =/  res=(unit (unit cage))  (rof ~ i.path u.bem)
+  ?~  res    ~
+  ?~  u.res  ~
+  =/  cast  .^(vase %cf (en-beam -.u.bem /[p.u.u.res]/mime))
+  `!<(mime (slam cast q.u.u.res))
```

Comment:
@belisarius222 how's this?

btw, I noticed that `+handle-scry` does something very similar, but it uses the `%cc` approach.